src/frontend/src/markdown-pages/about.md

```diff
@@ -0,0 +1,66 @@
+---
+path: '/about'
+title: 'About'
+---
+
+## Introduction
```

Comment:
Let's remove this line, and only have `About` at the top vs. `About\n\nIntroduciton`

Reply:
removed introduction heading